src/include/storage/write_ahead_log/log_io.h

```diff
@@ -98,41 +98,48 @@ class BufferedLogWriter {
    * update is only written out when the BufferedLogWriter is persisted.
    * @param data memory location of the bytes to write
    * @param size number of bytes to write
+   * @return number of bytes written
```

Comment:
You need to additionally explain why this is different from `size` param. How one should use this return field, etc.